swift/swift.py

```diff
@@ -73,7 +87,9 @@ def parse(cls: Type[T], kwargs: str) -> T:
     return cls(**json.loads(kwargs))
 
 
-def strinfo(info: AppInfo) -> str:
+# TODO(kangz12345): Method name looks too specific.
+# Change this to dumps() whenever we can rename this symbol.
+def strinfo(info: Serializable) -> str:
     """Returns a JSON string converted from the given info.
 
     This is just a convenience function for users not to import dataclasses and json.

```

Comment:
Now we use this function. so the docstring should be updated.

Reply:
Great point!